src/Accounts/Authentication/Utilities/AzConfigReader.cs

```diff
@@ -0,0 +1,48 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Common.Authentication.Abstractions;
+using Microsoft.Azure.Commands.Shared.Config;
+using Microsoft.Azure.PowerShell.Common.Config;
+
+namespace Microsoft.Azure.Commands.Common.Authentication.Utilities
+{
+    static public class AzConfigReader
+    {
+
+        public static IAzureSession Session
```

Comment:
```suggestion
        internal static IAzureSession Session
```
only expose public methods